include/crypto.h

```diff
@@ -356,4 +359,17 @@ extern uint8_t badFECF;
 extern uint32_t crc32Table[CRC32TBL_SIZE];
 extern uint16_t crc16Table[CRC16TBL_SIZE];
 
+// GF(2^4) field and logarithm tables
+static const uint8_t gf_exp[30] = {
+    1, 2, 4, 8, 3, 6, 12, 11, 5, 10, 7, 14, 15, 13, 9, 1,
+    2, 4, 8, 3, 6, 12, 11, 5, 10, 7, 14, 15, 13, 9
+};
+
+static const uint8_t gf_log[GF_SIZE] = {
+    0, 0, 1, 4, 2, 8, 5, 10, 3, 14, 9, 7, 6, 13, 11, 12
+};
+
+// Generator polynomial coefficients for g(x) = x^4 + a^3x^3 + ax^2 + a^3x + 1
+static const uint8_t gen_poly[RS_PARITY + 1] = {1, 8, 2, 8, 1};
+
 #endif // CRYPTO_H
```

Comment:
Everything 'looks' ok, and tests are passing.  But I don't know enough to say if this is correct.